core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -48,9 +48,9 @@ oppia.directive('audioTranslationBar', [
 
         $('.oppia-translation-tab').on('drop', function(evt) {
           evt.preventDefault();
-          if (evt.target.classList.contains('oppia-drop-area-message') &&
-            scope.dropAreaIsAccessible) {
-            files = evt.originalEvent.dataTransfer.files;
+          if ((<Element><any>evt.target).classList.contains(
+            'oppia-drop-area-message')  && scope.dropAreaIsAccessible) {
```

Comment:
Drop extra space before `&&`.

/cc @oppia/dev-workflow-team -- missing lint check?

Reply:
Lint is actually not working on this PR due to which this is missed. 

Done.